lib/bokken_web/controllers/admin/ninja/ninja_json.ex

```diff
@@ -1,13 +1,13 @@
-defmodule BokkenWeb.Admin.NinjaView do
-  use BokkenWeb, :view
+defmodule BokkenWeb.Admin.NinjaJSON do
+
 
   alias Bokken.Uploaders.Avatar
 
-  def render("index.json", %{ninjas: ninjas}) do
-    %{data: render_many(ninjas, __MODULE__, "ninja.json")}
+  def show(%{ninjas: ninjas}) do
```

Comment:
```suggestion
  def index(%{ninjas: ninjas}) do
```
An index is meant to render a collection, while a show is meant to render an item.